bin/hsforecast

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/home/tmp/aburke/miniconda3/envs/hail/bin/python
```

Comment:
Can you switch this back to /usr/bin/env python? Using your specific path won’t make it usable by others.